### DISCUSSION TITLE
How do I connect rivet to a local model?

### DISCUSSION THREAD

**Discussion by adp60** [Ironclad/rivet#154] [Q&A] [answered]
I have been using textgen webui + openai extension to run my flowise flows and would like to do the same on rivet.  How do I configure the options to enable this?  All I am getting is  404 errors on the  oogabooga console window.

**Comment by abrenneke** [comment]
Are you using the oobabooga plugin that HU$H developed? I'm afraid I can't help too much without knowing the exact errors you're getting and how you've set things up in Rivet. Any more information you can give would be appreciated.

**Comment by hushaudio** [comment, marked as answer]
https://github.com/hushaudio/rivet-oobabooga-plugin

**Comment by adp60** [comment]
I'm sorry, I didn't know about this plugin, until now. I have added it and it works like a charm. Thank you so much @hushaudio for this. I was attempting to connect it like langflow /flowise where you just input the api address and it works. Thank you for the assistance..

**Reply by hushaudio** [reply]
Yea for sure I’m glad it’s already getting used I just finished this version yesterday. If you have any feature requests let me know and I’ll see what’s possible. Looking into LoRa selection next to see if it’s supported in the api, and possibly add it to oobabooga if it’s not available.

**Reply by adp60** [reply]
Thanks for working on this..  My migration from flowise is now complete..  Is there like a link to all the plugins available at the moment? e.g I would like to search the web as part of a graph, same way agents work in langflow. Is this possible? TIA..

**Reply by abrenneke** [reply]
I don't think that plugin exists yet! Just the HTTP Call node that's built-in to Rivet. You should open an issue or post in the discord about a plugin that does that!
